cmd/ctr/commands/images/pull.go

```diff
@@ -132,8 +135,7 @@ command. As part of this process, we do the following:
 				fmt.Printf("image chain ID: %s\n", chainID)
 			}
 		}
-
-		fmt.Println("done")
+		fmt.Printf("done: %-4.1fs\t\n", time.Since(start).Seconds())
```

Comment:
How do you feel about:
```suggestion
		fmt.Printf("done: %s\t\n", time.Since(start))
```

So it'll choose proper formatting depending on elapsed time:
```
done: 10s	
done: 1m40s	
done: 2h46m40s
```
https://play.golang.org/p/jFRYNWE7UDE

Reply:
Thanks for the suggestion. It definitely looks better even though just using seconds makes it more script friendly. Updated the PR.